db/primary_migrate/20250106232958_drop_proofing_components_table.rb

```diff
@@ -0,0 +1,23 @@
+class DropProofingComponentsTable < ActiveRecord::Migration[7.2]
+  def change
+    drop_table :proofing_components do |t|
+      t.integer "user_id", null: false, comment: "sensitive=false"
```

Comment:
In case anyone else asks themselves, "Why does `drop_table` require columns when SQL obviously doesn't?"... The answer is that it's so the migration is reversible (it will create the table if undone).